core/templates/domain/topic/TopicObjectFactory.ts

```diff
@@ -76,6 +77,7 @@ export class Topic {
   _practiceTabIsDisplayed: boolean;
   _metaTagContent: string;
   _pageTitleFragmentForWeb: string;
+  _skillsForDiagnosticTest: ShortSkillSummary[];
```

Comment:
Should this be _skillSummariesForDiagnosticTest? Not sure.

Reply:
Updated